### PR TITLE
Translation Bugfixes

### PR DESCRIPTION
**Pull request by ovogel-wigital**
Added incorrect or not integrated translations.

### CODE REVIEW

File: code/Forms/AssetFormFactory.php [hunk 1]
```diff
@@ -360,7 +360,7 @@ protected function getFormFieldSecurityTab($record, $context = [])
         unset($editorsOptionsField['Anyone']);
 
         return Tab::create(
-            'Permissions',
+            'Permissions', _t(__CLASS__ . '.PERMISSIONS', 'Permissions'),
```
Comment (robbieaverill) [inline code/Forms/AssetFormFactory.php]:
```suggestion
            _t(__CLASS__ . '.PERMISSIONS', 'Permissions'),
```

File: code/Forms/FileFormFactory.php [hunk 2]
```diff
@@ -411,6 +431,7 @@ protected function getPopoverActions($record)
         $this->beforeExtending('updatePopoverActions', function (&$actions, $record) {
             // add the unpublish and replace file actions to the start of the array
             array_unshift($actions, $this->getUnpublishAction($record));
+            array_unshift($actions, $this->getDownloadFileAction($record));
```
Comment (robbieaverill) [inline code/Forms/FileFormFactory.php]: Is this a new feature? I can't see where this existed before.

**Comment by emteknetnz** [comment]
This pull request hasn't had any activity for a while. Are you going to be doing further work on it, or would you prefer to close it now?

**Comment by emteknetnz** [comment]
It seems like there's going to be no further activity on this pull request, so we've closed it for now. Please open a new pull-request if you want to re-approach this work, or for anything else you think could be fixed or improved.

**Comment by dhensby** [comment]
This looks like a fairly handy commit that just needs a small amount of work to get it over the line

**Pull request closed without merging**